toolkit/scripts/build_tag.mk

```diff
@@ -31,3 +33,8 @@ RELEASE_VERSION := $(RELEASE_VERSION)
 
 # Image tag - empty by default. Does not apply to the initrd.
 IMAGE_TAG          ?=
+
+# Image Customizer Version
+IMAGE_CUSTOMIZER_VERSION ?= 0.1.0
```

Comment:
Minor consideration - Consider splitting this out into separate variables aligned to the semantic versioning schema. Could help with future clarity that this the IMAGE_CUSTOMIZER_VERSION is following semantic versioning and prevent accidental input errors

Reply:
I expanded the code comment.